releasenotes/notes/0.25/dagcircuit-separable-circuits-142853e69f530a16.yaml

```diff
@@ -1,5 +1,5 @@
 ---
-features:
+features_circuits:
```

Comment:
I would probably put this under transpiler too, because at least in my mind the DAGCircuit api is for the transpiler and not really for circuit users.